test/fixtures/lightscript/arrow-methods/class-malformed-async-skinny-arrow/actual.js

```diff
@@ -0,0 +1,3 @@
+class X {
+  async method() -> await a
```

Comment:
Actually the reason I made this a non-getter test is, like the fat arrow getter comment, you also can't do it in JS and so it seemed obvious. And there's a `class-malformed-async-fat-arrow` but no skinny until now